src/eos.cpp

```diff
@@ -539,7 +539,7 @@ real struct_eos::density_at(real R, real dr) {
 	real r;
 	real h = h0();
 	real hdot = 0.0;
-	const int N = std::max(int(R / dr + 1.0), 32);
+	const int N = std::min(std::max(int(R / dr + 1.0), 2),10);
```

Comment:
Can this change be explained?
It seems that the maximal N in the new version is 10, and the minimal is 2 (instead of 32).